lib/ExpensiMark.js

```diff
@@ -174,7 +174,7 @@ export default class ExpensiMark {
 
                 // Replaces open and closing <br><br/> tags with a single <br/>
                 pre: inputString => inputString.replace('<br></br>', '<br/>').replace('<br><br/>', '<br/>'),
-                regex: /<br\s*[/]?>/gi,
+                regex: /<br\s*[/]?>\n?/gi,
```

Comment:
Can you update one of the tests to ensure this case is covered?

Reply:
Ok